jjb/dynamic/scale-ci-pipeline-bm-4.7.yml

```diff
@@ -0,0 +1,345 @@
+    - bool:
+        default: true
+        description: ''
+        name: CLUSTER_DENSITY
+    - bool:
+        default: true
+        description: ''
+        name: KUBELET_DENSITY
+    - bool:
+        default: false
+        description: ''
+        name: KUBELET_DENSITY_LIGHT
+    - bool:
+        default: false
+        description: ''
+        name: NS_PER_CLUSTER
+    - bool:
+        default: false
+        description: ''
+        name: PROMETHEUS_TEST
+    - bool:
+        default: false
+        description: ''
+        name: BYO_SCALE_TEST
+    - bool:
+        default: false
+        description: ''
+        name: LOGGING_SCALE_TEST
+    - bool:
+        default: false
+        description: ''
+        name: BASELINE_SCALE_TEST
+    - bool:
+        default: true
+        description: ''
+        name: UPERF
+    - bool:
+        default: false
+        description: ''
+        name: KRAKEN
+    - bool:
+        default: false
+        description: ''
+        name: SCALE_CI_MS_OSDE2E
+    - bool:
+        default: false
+        description: ''
+        name: SCALE_CI_MS_ROSA
+    - bool:
+        default: false
+        description: 'Caputers Prometheus DB and Cluster information using must-gather.'
+        name: SCALE_CI_DIAGNOSIS
+    - string:
+        default: http://dell-r510-01.perf.lab.eng.rdu2.redhat.com/perfscale_ci/properties/watcher/watcher.properties
+        description: ''
+        name: SCALE_CI_WATCHER_PROPERTY_FILE
+    - string:
+        default: http://dell-r510-01.perf.lab.eng.rdu2.redhat.com/perfscale_ci/properties/aws-next/build-triggers/trigger_4.7.properties
+        description: ''
+        name: SCALE_CI_BUILD_TRACKER_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: OPENSTACK_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: BROWBEAT_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: OPENSHIFTv3_PROPERTY_FILE
+    - string:
+        default: http://dell-r510-01.perf.lab.eng.rdu2.redhat.com/perfscale_ci/properties/aws-next/openshift_install.properties
+        description: ''
+        name: OPENSHIFTv4_ON_AWS_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: OPENSHIFTv4_ON_AZURE_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: OPENSHIFTv4_ON_GCP_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: OPENSHIFTv4_ON_OSP_PROPERTY_FILE
+    - string:
+        default: http://dell-r510-01.perf.lab.eng.rdu2.redhat.com/perfscale_ci/properties/baremetal/4.7/openshift_install.properties 
+        description: ''
+        name: OPENSHIFTv4_ON_BM_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: TOOLING_PROPERTY_FILE
+    - string:
+        default: ''
+        description: ''
+        name: CONFORMANCE_PROPERTY_FILE
+    - string:
```

Comment:
We might want to replace this with the baremetal properties file. Same for other workloads. Watcher will replace the pipeline with the links specified here every run.

Reply:
Ok they don't exist yet, so will just remove references for now and will add back when needed.